R/derive_var_dthcaus.R

```diff
@@ -219,6 +231,21 @@ derive_var_dthcaus <- function(dataset,
 #'
 #' @param date A character vector to be used for sorting `dataset`.
 #'
+#' @param date_imputation The value to impute the day/month when a datepart is missing.
+#'
+#'   If `NULL`: no date imputation is performed and partial dates are returned as missing.
+#'
+#'   Otherwise, a character value is expected, either as a
+#'   - format with day and month specified as 'mm-dd': e.g. '06-15' for the 15th
+#'   of June
+#'   - or as a keyword: 'FIRST', 'MID', 'LAST' to impute to the first/mid/last day/month.
+#'
+#'   Default is `NULL`
+#'
+#' @param order Sort order
+#'
+#'   Additional character vector to be used for sorting `dataset` to avoid duplicate record warning.
```

Comment:
I would change to
```
Additional variables to be used for sorting `dataset` to avoid duplicate record warning.

*Default*: `NULL`

*Permitted Values*: list of variables or `desc(<variable>)` function calls
  created by `vars()`, e.g., `vars(ADT, desc(AVAL))` or `NULL`
```
to be more precise.

Reply:
updated

Reply:
Could you replace "character vector" with "variables"? I think "character vector" is misleading.